backend/db_orm/table_classes.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any, Union
+
+from sqlalchemy import Column, Integer, String, ForeignKey, Boolean, DateTime, PrimaryKeyConstraint # type: ignore
+from sqlalchemy.ext.declarative import declarative_base  # type: ignore
+
+SHORT_STRING_LEN = 200
+LONG_STRING_LEN = 2000
+
+Base: Any = declarative_base()
+
+
+class User(Base):
+    __tablename__ = 'user'
+
+    username: Column = Column(String(SHORT_STRING_LEN), primary_key=True)
+    password_hash: Column = Column(String(SHORT_STRING_LEN))
+    role_type: Column = Column(Boolean)
+    update_time: Column = Column(DateTime)
+
+
+class Laborer(Base):
+    __tablename__ = 'laborers'
+
+    username: Column = Column(String(SHORT_STRING_LEN), ForeignKey(User.username), primary_key=True)
+    parent_username: Column = Column(String(SHORT_STRING_LEN))
+    first_name: Column = Column(String(SHORT_STRING_LEN))
+    last_name: Column = Column(String(SHORT_STRING_LEN))
+    gender: Column = Column(Integer)
+    phone_number: Column = Column(String(SHORT_STRING_LEN))
+    address: Column = Column(String(LONG_STRING_LEN))
+    aadhar_card_status: Column = Column(Boolean)
+    open_for_work: Column = Column(Boolean)
+
+class Contractor(Base):
+    __tablename__ = 'contractors'
+
+    username: Column = Column(String(SHORT_STRING_LEN), ForeignKey(User.username), primary_key=True)
+    parent_username: Column = Column(String(SHORT_STRING_LEN))
+    first_name: Column = Column(String(SHORT_STRING_LEN))
+    last_name: Column = Column(String(SHORT_STRING_LEN))
+    gender: Column = Column(Integer)
+    phone_number: Column = Column(String(SHORT_STRING_LEN))
+    address: Column = Column(String(LONG_STRING_LEN))
+
+
+class Skill(Base):
+    __tablename__ = 'skills'
+
+    id: Column = Column(Integer, primary_key=True)
+    name: Column = Column(String(SHORT_STRING_LEN))
+
+
+class Location(Base):
+    __tablename__ = 'locations'
+
+    id: Column = Column(Integer, primary_key=True)
+    name: Column = Column(String(SHORT_STRING_LEN))
+
+
+class LaborerPreferredLocation(Base):
+    __tablename__ = 'laborer_preferred_locations'
+
+    username: Column = Column(String(SHORT_STRING_LEN), ForeignKey(User.username))
+    skill_id: Column = Column(Integer, ForeignKey(Skill.id))
```

Comment:
I think this and below table are interchanged, plus this table needs more column and I can not find Location table which has State,city and district as primary key. either I am missing something or that has been missed. check the deleted create.sql file for more reference, so this makes me think were all the flows tested ?

Reply:
> I think this and below table are interchanged, plus this table needs more column and I can not find Location table which has State,city and district as primary key. either I am missing something or that has been missed. check the deleted create.sql file for more reference, so this makes me think were all the flows tested ?

found location table, but need to add more column to it